Sources/Test.swift

```diff
@@ -0,0 +1,8 @@
+import Foundation
+
+func test() {
+    var test = FileAttributeKey.creationDate
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/shogo4405/HaishinKit.swift/blob/7fedc52c24ef08832a21716080a8697f62f01528/Sources/Test.swift#L4"></span>The 'creationDate' is the require reason api. Please check privacy manifest files.</td>
    </tr>
  </tbody>
</table>